qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -232,9 +244,9 @@ def _batch_last(m: Tensor) -> Tensor:
         )
 
 
-class PyQObservable(Module):
+class PyQObservable(pyq.QuantumCircuit):
     def __init__(self, block: AbstractBlock, n_qubits: int, config: Configuration = None):
-        super().__init__()
+        super().__init__(n_qubits, [])
```

Comment:
I'm a bit scared of this init with `[]`, are we sure it's ok?

mutable defaults (as I think this might be regarded) is scary

Reply:
redone, you can check it tomorrow @awennersteen